src/styles/styles.js

```diff
@@ -3779,6 +3779,13 @@ const styles = (theme) => ({
         lineHeight: variables.lineHeightLarge,
     },
 
+    aspectRatioLottie: (source) => {
+        if (typeof source === 'object' && !source.uri) {
+            return {aspectRatio: source.w / source.h};
+        }
+        return {aspectRatio: '1'};
```

Comment:
I prefer setting this undefined and let user fully customize style properties.
If both width and height are defined, they're skipped because aspectRatio overwrites them.
```suggestion
        return {};
```